roles/KubevirtCommonTemplatesBundle/templates/kubevirt-base-image-roles.yaml.j2

```diff
@@ -0,0 +1,37 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Did you make sure no other permissions are required for CDI? Does CDI's service account require any special permissions as well? If so, how would we bind the role to that service account?